bin/create-component.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# $1 = the first argument in the command line
+
+if [ -z "$1" ]
+  then
+    echo "Please enter a component name"
+    exit 1;
+  else
+    transforms component name from PascalCase to kebab-case
```

Comment:
```suggestion
    # transforms component name from PascalCase to kebab-case
```

Is this meant to be a comment?

Reply:
oops, yep